PartnerSdk/src/main/java/com/microsoft/store/partnercenter/models/relationships/PartnerRelationshipType.java

```diff
@@ -13,9 +13,9 @@
  */
 public enum PartnerRelationshipType 
 {
-    @JsonProperty("is_indirect_reseller_pf")
-    IsIndirectResellerOf,
+    @JsonProperty("is_indirect_reseller_of")
```

Comment:
>is_indirect_reseller_of [](start = 19, length = 23)

Thanks for such catches. Didn't realize we had such typo based errors in the codebase :(. :sunglasses: